include/sound/sof/info.h

```diff
@@ -111,7 +111,7 @@ struct sof_ipc_cc_version {
 
 	char name[16]; /* null terminated compiler name */
 	char optim[4]; /* null terminated compiler -O flag value */
-	char desc[]; /* null terminated compiler description */
+	char desc[32]; /* null terminated compiler description */
```

Comment:
we really should replace all `char` with `uint8_t` in ABI types - in a separate patch or patch series

Reply:
@lyakh Ouch, yes indeed, I hadn't noticed that. But yes, this is not safe packing. But not for this PR.

Reply:
i'd like this to be fixed as well, better send one update upstream.